GFS_layer/GFS_physics_driver.F90

```diff
@@ -1451,6 +1484,10 @@ subroutine GFS_physics_driver                         &
       Diag%u1(:)      = Statein%ugrs(:,1)
       Diag%v1(:)      = Statein%vgrs(:,1)
       Sfcprop%qsfc(:) = qss(:)
+        
+      ! KGao
+      Diag%hflx(:) = hflx(:)
```

Comment:
How do these differ from the existing diagnostics? Is this exposing the behavior of the coupler itself rather than the flux "seen" by the atmosphere?

Reply:
These are two new diagnostics, which were to ensure the fields directly used by the SHiELD physics are identical to what in the coupler.